.github/workflows/codesee-arch-diagram.yml

```diff
@@ -0,0 +1,23 @@
+# This workflow was added by CodeSee. Learn more at https://codesee.io/
```

Comment:
I don't think this file belongs to this PR. Seems the tool you are using added it automatically...

Reply:
removed